admin-manual/maintenance/common-atom-queries.rst

```diff
@@ -0,0 +1,2000 @@
+
+In the above example query: 
+
+* ``source-repo-slug`` represents the slug of the :term:`archival institution` currently
+  associated with the descriptions, and
+* ``target-repo-slug`` represents the slug of the archival institution to which you
+  would like to move your descriptions.
+
+.. IMPORTANT::
+
+   Remember, after making data changes, you should restart PHP-FPM, clear the
+   application cache, and re-populate your search index to ensure that all
+   changes show up properly in AtoM. See: 
+
+   * :ref:`troubleshooting-restart-php-fpm`
+   * :ref:`maintenance-clear-cache`
+   * :ref:`maintenance-populate-search-index`   
+
+:ref:`Back to the top <common-atom-queries>`
+
+.. _sql-change-levels-io:
+
+Bulk change levels of description
+---------------------------------
+
+If you wanted to change, for example, all "Item" level descriptions in your AtoM 
+installation to "Part" levels, we can use SQL to accomplish this. First, 
+we will need the term ID values for both the current (e.g. Item) and target 
+(e.g. Part) levels of description. See the last example in this section: 
+
+* :ref:`sql-term-id`
+
+In the following example query, we've learned that Part has an ID of 285, and 
+Item has an ID of 227. We can now update all :term:`archival description` 
+records with an "Item" level of description to "Part" with the following: 
+
+.. code-block:: mysql
+
+   UPDATE information_object SET level_of_description_id=285 WHERE level_of_description_id=227;
+
+:ref:`Back to the top <common-atom-queries>`
+
+.. _sql-delete-translations:
+
+Delete translations
+-------------------
+
+In AtoM, content can be translated into multiple languages - see: 
+
+* :ref:`translate-content`
+
+We can see a count of how many descriptions we have in a specific culture, using
+the ISO 639-1 two-letter culture codes as a parameter to perform a count on the 
+``information_object_i18n`` table, where :term:`archival description` fields 
+that are translatable are stored. For example, to see a count of all 
+descriptions in Catalan: 
+
+ .. code-block:: mysql
+
+    SELECT COUNT(*) FROM information_object_i18n WHERE culture='ca';
+
+We can also delete all translations for a specific culture. 
+
+.. WARNING:: 
+
+   Make sure this is what you want to do! As always we strongly recommend that 
+   you make a :ref:`database backup <cli-backup-db>` before you proceed. 
+
+For example, to delete all translations in French: 
+
+.. code-block:: mysql
+
+   DELETE FROM information_object_i18n WHERE culture='fr';   
+
+If you only want to delete a specific description's translation, we can do so 
+by using the :term:`slug` of the target description. In the following example
+we are deleting Spanish translations from a description with the slug ``test-01``: 
+
+.. code-block:: mysql
+
+   DELETE io.* FROM information_object_i18n io 
+   JOIN slug ON slug.object_id = io.id
+   WHERE (io.culture = "es")
+   AND (slug.slug = "test-01");
```

Comment:
`DELETE io` would be enough, no need for any parenthesis and same issue than above for the related resources.